elf_load.c

```diff
@@ -198,11 +198,20 @@ static int elf_load_parse(struct elf_prog *elf_prog, Elf *elf)
 				elf_prog->upperl = phdr.p_paddr + phdr.p_memsz;
 		}
 		UK_ASSERT(elf_prog->lowerl <= elf_prog->upperl);
+
+		if (phdr.p_offset <= ehdr.e_phoff &&
```

Comment:
Perhaps a comment like "Calculate the in-memory phdr offset" above this check?

Reply:
👌